examples/MLIRVectorGPU/makefile

```diff
@@ -0,0 +1,1022 @@
+#!/bin/bash
+BUDDY_OPT := ../../build/bin/buddy-opt
```

Comment:
Please use the variable definition like [this](https://github.com/buddy-compiler/buddy-mlir/blob/main/examples/BuddyMatmul/makefile#L2) in case of modifications when changing the `build` path.